api/mailer/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+  This module is used for sending emails
+"""
+
+import os
+from flask_mail import Mail, Message  # type: ignore
+from jinja2 import Template
+from flask import Flask
+
+FRONTEND_IP = os.getenv("FRONTEND_IP")
+
+
+def send_invite_email(app: Flask, token: str, email: str) -> None:
+    """
+    Prepare the admin invite email and send it
+
+    Parameters:
+
+      app: the flask app
+      token: token created for the email
+      email: address the email is being sent to
+
+      Returns:
+        None
+    """
+
+    mail = Mail(app)
+    with open("api/mailer/templates/invite_template.txt", "r") as f:
+        template = Template(f.read())
```

Comment:
Isn't there a way to serve up a jinja template rather than serving a rendering text file?

Reply:
I'll look into the template